stock_change_qty_reason/models/stock_inventory_line_reason.py

```diff
@@ -0,0 +1,11 @@
+# Copyright 2019 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class StockInventoryLineReason(models.Model):
+
+    _name = 'stock.inventory.line.reason'
```

Comment:
@hveficent I would have added an active field to deactivate old ones without losing values (+ active toggle button).